docs/fastdds/security/auth_plugin/auth_plugin.rst

```diff
@@ -190,6 +190,10 @@ However, all other commands are only compatible with Linux OS.
     commonName = eProsima Main Test CA
     emailAddress = mainca@eprosima.com
 
+    [root_ca_extensions]
+    basicConstraints = critical, CA:true
+    keyUsage = critical, keyCertSign, cRLSign
```

Comment:
I have only been able to test the `CA:true` extension. I don't know much about the other ones you have included, but we should make sure the certificates generated with this configuration will work fine, could you please test that if you haven't already?